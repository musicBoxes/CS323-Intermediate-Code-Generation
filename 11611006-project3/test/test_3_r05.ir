FUNCTION foo :
PARAM i
PARAM j
PARAM k
dyj0 := i - j
dyj1 := k * #2
dyj2 := dyj0 + dyj1
RETURN dyj2
FUNCTION main :
a := #110
b := #97
c := #3
ARG c
ARG b
ARG a
dyj := CALL foo
c := dyj
WRITE c
RETURN #0
