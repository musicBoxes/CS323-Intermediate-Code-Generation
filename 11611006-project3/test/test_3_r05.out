FUNCTION foo :
PARAM i
PARAM j
PARAM k
dyj0 := i - j
dyj1 := k * #2
dyj2 := dyj0 + dyj1
RETURN dyj2
FUNCTION main :
a := dyj
b := dyj
c := dyj
c :=
RETURN #0
